### ISSUE TITLE
Breaking on server-side when requiring the module

### ISSUE THREAD

**Issue by d-oliveros** [AlexKvazos/VanillaToasts#3]
The application is breaking when requiring the module on server-side. Isomorphic applications won't be using this on the server, but will most likely require the module on server-side.

```
ReferenceError: document is not defined
    at /home/david/careerdean/main/node_modules/vanillatoasts/vanillatoasts.js:12:7
    at /home/david/careerdean/main/node_modules/vanillatoasts/vanillatoasts.js:4:22
    at Object.<anonymous> 
    ...
(/home/david/careerdean/main/node_modules/vanillatoasts/vanillatoasts.js:9:3)
```

**Comment by ColtraneNadler** [comment]
Might pick this up as well.

**Comment by d-oliveros** [comment]
Fixed in #6
